extensions/arc/src/ui/dashboards/postgres/postgresOverviewPage.ts

```diff
@@ -154,7 +155,13 @@ export class PostgresOverviewPage extends DashboardPage {
 				try {
 					const password = await promptAndConfirmPassword(input => !input ? loc.enterANonEmptyPassword : '');
 					if (password) {
-						// TODO: azdata arc postgres server edit --admin-password
+						await this._azdataApi.azdata.arc.postgres.server.edit(
+							this._postgresModel.info.name,
+							{
+								adminPassword: true,
+								noWait: true
```

Comment:
@bergeron Is there a specific reason you want the noWait here? In general I would think we should wait for the command to finish so that if an error occurred we can expose it to the user.

Reply:
For other edits yes.  Some edits require downtime and we'd want to wait for the resource to become ready.  But password reset doesn't have detection that it has finished propagating to all nodes, so there's nothing to wait for.  (It will have this detection in the future when we do it via a task resource that can be checked for completion)

Reply:
Is there any reason why we wouldn't just remove the noWait option for now and then when that functionality is hooked up in the future we wouldn't need to do anything ourselves to have that functionality in ADS too?

Reply:
We shouldn't always omit --no-wait because waiting adds a minimum 5 second delay.  The CLI needs to become smarter about when an edit has been processed by the controller.  Waiting 5 seconds and polling the state isn't sufficient for several reasons.  If anything we'd always pass --no-wait until it's smarter.

Reply:
Oh interesting - is that just a thing for the edit command? Yeah if that's the case that sounds reasonable - although I agree that's less than desirable behavior.

Reply:
The way it's supposed to work, is that any command that takes time to complete waits by default.  And offers --no-wait to return immediately.  Currently that's create, edit, and take a backup.  Edit is the only one where the detection needs improvement.